### DISCUSSION TITLE
How to use the extracted images

### DISCUSSION THREAD

**Discussion by cyhasuka** [infiniflow/ragflow#1440] [General]
I found in README.md : 2024-06-27 Supports extracting images from Docx files.
However, the images seem to be merely extracted and not inserted into LLM's replies, and I'd like to allow LLM to use them like normal chat. This feature seems to be easily accomplished in the form of a markdown link. Are there any plans for this?

**Comment by KevinHuSh** [comment]
Applying it In format of markdown link is a very good idea.
Right now, images are displayed by hovering the little exclamation mark.

**Reply by cyhasuka** [reply]
Looking forward to applying this as a markdown link, as many times it is not desirable to show the knowledge base source.
